ground_control/GroundControl.gd

```diff
@@ -447,7 +447,40 @@ func expert_training():
 			#if false shows an alert 
 			$CanvasLayer/Alert.visible = true
 
+func info_in(button_id):
+	match button_id:
+		UpgradeType.OXYGEN: 
+			info.text = "Each point gives " + str(Global.OXYGEN_STEP) + " more oxygen."
+		UpgradeType.HEALTH:
+			info.text = "Each point gives " + str(Global.HEALTH_STEP) + " more health."
+		UpgradeType.ATTACK:
+			info.text = "Increases your damage output."
+		UpgradeType.ACCURACY:
+			info.text = "Improves your chance to hit with all abilities."
+		UpgradeType.DEFENSE:
+			info.text = "Reduces incoming damage, and increases healing."
+		UpgradeType.SPEED:
+			info.text = "Reduces the wait time between actions."
+		UpgradeType.EVADE:
+			info.text = "Increases the chance of not getting hit."
+		UpgradeType.COMBATTRAINING:
+			info.text = 'Gives you the ability "Basic Swing."'
+		UpgradeType.BASICWEAPONS:
+			info.text = "Gives you a random floor one attack bonus item."
+		UpgradeType.BASICDEFENSE:
+			info.text = "Gives you a random floor one defense bonus item."
+		UpgradeType.ADVANCETRAINING:
+			info.text = 'Gives you the ability "Solid Swing."'
+		UpgradeType.ADVANCEWEAPONS:
+			info.text = "Gives you a random floor two attack bonus item."
+		UpgradeType.ADVANCEDEFENSE:
+			info.text = "Gives you a random floor two defense bonus item."
+		UpgradeType.EXPERTTRAINING:
+			info.text = 'Gives you the ability "Flawless Swing."'
```

Comment:
This can be reduced further by creating a const array at the top with each description in order by the enumerator.

Then in the function you just need to make sure the parameter is greater than zero, and less than the last item in the enum, which you can do by adding this to the enum:
`EXPERTTRAINING,`
`LAST_UPGRADE_TYPE = EXPERTTRAINING,`

It's not required for this PR, but it would cut down on the function size.
Let me know if you want to walk through it.